cmd/kubeadm/app/cmd/init.go

```diff
@@ -248,6 +251,14 @@ func AddInitOtherFlags(
 		uploadCerts, options.UploadCerts, *uploadCerts,
 		"Upload certfificates to kubeadm-certs secret.",
```

Comment:
let's change this one too for consistency:
`Upload control-plane certificates to the kubeadm-certs Secret.`